src/entry.js

```diff
@@ -62,7 +62,7 @@ module.exports = (argv) => {
 
   if (!utils.isValidAppInstall(command)) {
     console.error(colors.red(
-      `Looks like your package.json is missing ${PLATFORM_PACKAGE} or you haven't run npm install yet!`
+      `Looks like your package.json is missing \`${PLATFORM_PACKAGE}\`, its version is a range, or you haven't run npm install yet!`
```

Comment:
It seems clear to me. Each of the things in the list (missing package, version is a range, haven't installed) is a bad thing that's keeping the cli from working. It might be more clear if all 3 were negative, eg `version isn't a single version` or similar. Is that better?